modules/network-observability-netobserv-cli-reference.adoc

```diff
@@ -0,0 +1,169 @@
+
+| `cleanup`
+| Remove netobserv components.
+
+| `version`
+| Print the software version.
+
+| `help`
+| Show help.
+|===
+
+== Feature commands
+The Network Observability Operator features can only be enabled when capturing flows. They do not work with packet capture.
+
+.Syntax
+[source,terminal]
+----
+$ oc netobserv flows [<feature_command>] [<subcommands>]
+----
+
+.Network Observability feature enablement commands
+|===
+|Option| Description| Possible values| Default
+
+| `--help`
+| Show help
+| N/A
+| N/A
+
+| `--enable_pktdrop` 	
+| Enable packet drop
+| `true, false`
+| `false`
+
+| `--enable_rtt`
+| Enable round trip time 	
+| `true`, `false` 	
+| `false`
+
+| `--enable_dns`
+| Enable DNS tracking 	
+| `true`, `false` 	
+| false
+
+| `--interfaces`
+| Interfaces to match on the flow e.g., "eth0,eth1" 	
+| ""
+|===
+
+== Flow capture subcommands
+Flow capture has mandatory commands and optional commands. 
+
+.`oc netobserv flows` syntax
+[source,terminal]
+----
+$ oc netobserv flows [<feature_command>] [<subcommands>]
+----
+
+.Flow capture subcommands
+|===
+|Option| Description| Possible values| Mandatory| Default
+
+| `--enable_filter`
+| Enable flow filter
+| `true`, `false` 	
+| yes 
+| false
+
+| `--action`
+| Action to apply on the flow
+| `Accept`, `Reject`
+| yes
+| Accept
+
+| `--cidr`
+| CIDR to match on the flow. for example `1.1.1.0/24`, `1::100/64`, or `0.0.0.0/0` 
+| yes
+| `0.0.0.0/0`
+
+| `--protocol`
+| Protocol to match on the flow
+| TCP, UDP, SCTP, ICMP, ICMPv6 	
+| no
+| `TCP`
+
+| `--direction`
+| Direction to match on the flow
+| `Ingress`, `Egress`
+| no
+
+| `--dport`
+| Destination port to match on the flow, for example `80`, `443`, or `49051`
+| no
+
+| `--sport`
+| Source port to match on the flow, for example `80`, `443`, or `49051`
+| no
+
+| `--port`
+| Port to match on the flow, for example `80`, `443`, or `49051`
+| no 
+|
+
+| `--sport_range`
+| Source port range to match on the flow, for example `80-100` or `443-445`
+| no 	
+| 
+
+| `--dport_range` 	
+| Destination port range to match on the flow, for example `80-100`
+| no
+|
+
+| `--port_range`
+| Port range to match on the flow, for example `80-100` or `443-445`
+| no
+|
+
+| `--icmp_type`
+| ICMP type to match on the flow, for example `8` or `13`
+| no
+|
+
+| `--icmp_code`
+| ICMP code to match on the flow, for example `0` or `1`
+| no
+|  	
+
+| `--peer_ip`
+| Peer IP to match on the flow, for example `1.1.1.1` or `1::1`
+| no 	
+|
+|===
+
+== Packet capture subcommand
+You can filter on port and protocol for packet capture data. 
+
+.`oc netobserv packets` syntax
+[source,terminal]
+----
+oc netobserv packets [<subcommand>]
+----
+.Packet capture subcommand
+` --filter`:: Enable packet capture filtering. Options for packet capture filtering are protocol (tcp, udp) and port. You can specify filtering options using a comma as delimeter. For example `--filter=tcp,80` specifies the `tcp` protocol and port `80`.
+
```

Comment:
There's a random `in` at EOF. Dont see it here😂
<img width="790" alt="Screenshot 2024-05-16 at 11 07 01 AM" src="https://github.com/openshift/openshift-docs/assets/65666095/ce18a4cc-4f47-4768-aaae-38706a41c27b">

Reply:
Oh I see, it was in the Assembly file that the reference module is included in. It looks like I started typing "include" there or something.

Reply:
🤦‍♀️ I'm sorry I'm going too fast between PRs 😓